### ISSUE TITLE
Adding contact us page

### ISSUE THREAD

**Issue by fatma-sia27** [avinashkranjan/Amazing-Python-Scripts#812]
Hey @avinashkranjan , if a contact us form will be there in the website then it would be great. Isn't it?
If you find this issue meaningful then please assign it to me so that I can make a contact us form for this website.
Thank you

**Comment by kaustubhgupta** [comment]
Hi @fatma-sia27,
The idea of the contact form is nice. Can you let me know how the details entered by user will be managed? Stored somewhere or pushed to a database? Maybe firebase?

**Comment by fatma-sia27** [comment]
> Hi @fatma-sia27,
> The idea of the contact form is nice. Can you let me know how the details entered by user will be managed? Stored somewhere or pushed to a database? Maybe firebase?

Actually at present , Iam learning Web-dev, I had learned HTML amd CSS(almost) but backend is still left . So I meant to sat that I will make only frontend part.

**Comment by kaustubhgupta** [comment]
@fatma-sia27 okay no issues, will create a separate issue for that, You can start working on this. Refer to this directory for website files. All the website files are present in the gh-pages branch of this repo. Fork that branch and not the master branch. 

https://github.com/avinashkranjan/Amazing-Python-Scripts/tree/gh-pages

**Comment by fatma-sia27** [comment]
> @fatma-sia27 okay no issues, will create a separate issue for that, You can start working on this. Refer to this directory for website files. All the website files are present in the gh-pages branch of this repo. Fork that branch and not the master branch.
> 
> https://github.com/avinashkranjan/Amazing-Python-Scripts/tree/gh-pages

Ok thank you , I will do my best.

**Comment by fatma-sia27** [comment]
> @fatma-sia27 okay no issues, will create a separate issue for that, You can start working on this. Refer to this directory for website files. All the website files are present in the gh-pages branch of this repo. Fork that branch and not the master branch.
> 
> https://github.com/avinashkranjan/Amazing-Python-Scripts/tree/gh-pages

I should fork only gh page branck right?

**Comment by kaustubhgupta** [comment]
> > @fatma-sia27 okay no issues, will create a separate issue for that, You can start working on this. Refer to this directory for website files. All the website files are present in the gh-pages branch of this repo. Fork that branch and not the master branch.
> > https://github.com/avinashkranjan/Amazing-Python-Scripts/tree/gh-pages
> 
> I should fork only gh page branck right?

yes

**Comment by kaustubhgupta** [comment]
@fatma-sia27 updates on this?

**Comment by fatma-sia27** [comment]
> @fatma-sia27 updates on this?

Yes @kaustubhgupta I had made contact us page but since I am new to open source so I had made some mistakes during my previous and first PR so I am learning properly how to make PR . After that I will make PR.
Thank you

**Comment by kaustubhgupta** [comment]
> > @fatma-sia27 updates on this?
> 
> Yes @kaustubhgupta I had made contact us page but since I am new to open source so I had made some mistakes during my previous and first PR so I am learning properly how to make PR . After that I will make PR.
> Thank you

Okay, if face any issues then refer to my this comment where I have explained the whole procedure:
https://github.com/avinashkranjan/Amazing-Python-Scripts/issues/279#issuecomment-805558830

**Comment by fatma-sia27** [comment]
Okay @kaustubhgupta I will do it shortly. One more thing when I cloned this gh page branch then there are multiple files so I should push all the files or only "contact us" folder in which I had my code of "contact us" page?

**Comment by kaustubhgupta** [comment]
> Okay @kaustubhgupta I will do it shortly. One more thing when I cloned this gh page branch then there are multiple files so I should push all the files or only "contact us" folder in which I had my code of "contact us" page?

When you will enter this command:
```bash
git add .
``'
only the files you modified/created will be added to the stagging area. To confirm that you have changed only the required files, run:
```bash
git status
```
and you will be presented with a list of files that you changed/added.

**Comment by fatma-sia27** [comment]
hey @kaustubhgupta , I had created PR #841, I hope I made it correctly. Please review it and let me know if any changes required.
Thank you.

**Comment by fatma-sia27** [comment]
> > Okay @kaustubhgupta I will do it shortly. One more thing when I cloned this gh page branch then there are multiple files so I should push all the files or only "contact us" folder in which I had my code of "contact us" page?
> 
> When you will enter this command:
> 
> ```shell
> git add .
> ```
> 
> only the files you modified/created will be added to the staging area. To confirm that you have changed only the required files, run:
> 
> ```shell
> git status
> ```
> 
> and you will be presented with a list of files that you changed/added.

Hey @kaustubhgupta all the process will be as explained here only I should change branch when I compare and pull request.
Right?

**Comment by kaustubhgupta** [comment]
> > > Okay @kaustubhgupta I will do it shortly. One more thing when I cloned this gh page branch then there are multiple files so I should push all the files or only "contact us" folder in which I had my code of "contact us" page?
> > 
> > 
> > When you will enter this command:
> > ```shell
> > git add .
> > ```
> > 
> > 
> > only the files you modified/created will be added to the staging area. To confirm that you have changed only the required files, run:
> > ```shell
> > git status
> > ```
> > 
> > 
> > and you will be presented with a list of files that you changed/added.
> 
> Hey @kaustubhgupta all the process will be as explained here only I should change branch when I compare and pull request.
> Right?

yes

**Comment by fatma-sia27** [comment]
![img](https://user-images.githubusercontent.com/77666526/113990786-72380d80-986f-11eb-976a-ac5da33af700.jpeg)
which branch should I choose?

**Comment by fatma-sia27** [comment]
@kaustubhgupta 

> ![img](https://user-images.githubusercontent.com/77666526/113990786-72380d80-986f-11eb-976a-ac5da33af700.jpeg)
> which branch should I choose?

@kaustubhgupta

**Comment by fatma-sia27** [comment]
![WhatsApp Image 2021-04-08 at 1 41 39 PM](https://user-images.githubusercontent.com/77666526/113991801-6e58bb00-9870-11eb-8f92-357f7b0e2d8f.jpeg)
I am doing in correct branch ??

**Comment by fatma-sia27** [comment]
> ![WhatsApp Image 2021-04-08 at 1 41 39 PM](https://user-images.githubusercontent.com/77666526/113991801-6e58bb00-9870-11eb-8f92-357f7b0e2d8f.jpeg)
> I am doing in correct branch ??

@kaustubhgupta

**Comment by kaustubhgupta** [comment]
> ![WhatsApp Image 2021-04-08 at 1 41 39 PM](https://user-images.githubusercontent.com/77666526/113991801-6e58bb00-9870-11eb-8f92-357f7b0e2d8f.jpeg)
> I am doing in correct branch ??

yes, create the PR

**Comment by fatma-sia27** [comment]
@kaustubhgupta I had created PR #849 , Please review it.
 thank you

**Comment by srishtij2000** [comment]
Can I work on this issue?
